src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,18 @@
+import { Link } from "react-router-dom"
+
+function ArticleCard({article}) {
+  return (
+    <><li className='card' key={article.article_id}> 
```

Comment:
This fragment isn't doing anything as the li is a parent of all the other elements

Reply:
I recommend using prettier to format your code before PRs